omnibus/files/server-ctl-cookbooks/infra-server/attributes/default.rb

```diff
@@ -319,8 +335,8 @@
 default['private_chef']['opscode-erchef']['search_batch_max_size'] = '5000000'
 default['private_chef']['opscode-erchef']['search_batch_max_wait'] = '10'
 default['private_chef']['opscode-erchef']['search_auth_enabled'] = false
-default['private_chef']['opscode-erchef']['search_auth_username'] = 'admin'
-default['private_chef']['opscode-erchef']['search_auth_password'] = 'admin'
+default['private_chef']['opscode-erchef']['search_auth_username'] = "admin"
+default['private_chef']['opscode-erchef']['search_auth_password'] = "admin"
```

Comment:
the default password can probably be something else?

Reply:
check if these belong under opscode-erchef or opensearch hash.

Reply:
For external and internal, We expect the username & password for connecting to Opensearch.
External Scenario - We will not have more control on their opensearch setup(like updating the security config files for adding the secrets generated username & password).So like postgres We except the credentials from the customer.
Internal Scenario - We will be having the access to opensearch config files so we can add the generated username & password in the Security configs.
In short, Credantials should be passed in the external case but not in the internal.

Yes, We don't need these defaults set in the config file.And Preflight checks error out if they set search_provider to opensearch without setting 'search_auth_username' & 'search_auth_password'.